python/LLM Examples/Removing MNPI via Redaction for LLMs.ipynb

```diff
@@ -0,0 +1,569 @@
57Rlwyb9x48v19lnnQXtRigWg64BLHN86HXNRMp24ydeflFLHn9U8+fPRwc+qEfve1DFWTkcIfBo9NsMIE6av0BPvfCs7nvgAc2YMUM33XSTHn/oYWUDOB7cv43mX/MSOBG+5o0MNe/rSQHHiDDJAwzIIQyCMY1sRvYCYnAMp+UMi8XajYgiIqKhHrn3DPebMXNgWQfZIL+muc4a6xVLQrdIUAbHY7A4f/i9uZqGEGBeiVThitVvn20i92N9EWRITSgyJsrqsc3KJRqNCVcEYQ39FluRl+N77J+hzZ5Lv1KsrkNf9iN4mCvWW/vN3JvTU5KxHBE50bCi/P0pDuyyfpllqHlMWf8G/dPJWRdoT1ImHlp7yLOWYb5R0qypeE5OX6xqhIJk5DQTLonQgsu4AUUzHWBXzm9e8qd1sllFxBLeg+vCEcIsrA37orpbewG14tWDlSoRKukzIS/riYnMhy7oYqFZOgF8ws27D2uLqMQskWaOzI7koUsEKGsBPEI6M8M2KxYSxiox2tlnC53i6GjAoes8F9nhZ7/Zwww9NYLYj8bgDtHJmZfyrTHMFtB1EEFwP4Bn1BjCh2Yn6YOV6zR6OMIW16z87AvAr7EuVEsZgpGBbparME7tGpmV63LQ4VHOZIlXG33KxSrF+P4ZY+L10N1nOhdu28Dt0cF3j2a6udHAd7EpGfSZYwGamVBhoLGhnZyv0CABwbUGr7I8XXTeLJUUcj33IJe5EqlRzkM0mlvC+wbV3NyEUMic42brm11nY9Pe3YonYzpjHw3w3azcpBh3+OKRr4ceulg9jJu1z+aRCcM2T3z+EcqBZL18jsAdcYA8iYOEjmHmQz4udJPO6AyDDq2jaZj1pxKwKyYfwZSEhz2EKYjjh34OygY89cKJ2YpDH+FRmUuAyv2EjjFrUXuevaxOL2FjErnArslkDi255zwRkdMNry0ceAUYepQA/nEq5LOpIMJisAburVAHuRejqfeyKxdqfJGUyiK09ptYEg2wm9DOc+mfdTaCNdzT1aYE4sq+9v46JcVO17HzpEsuKtbZKlYzkxB8Wqs+k/6yciPTh7mI9XUEtI2hHYUoI+757ZkuzBHNcG3iza3tKE137aX5hDiC0eqp0KjRw2CoyPNYj1ID72MwaZIZ0zrG3cSoLuocjEhhfZDjkCv9Ng9CJUSBEAVCFAhRIESBEAVCFAhRIESB/wEFzKLcFOjY28FjYpS2B2NAPANnjovByBOwamMnIGKMA8yI8KXiQqmkJEkjpiTptgfWIYflYIMX58Ar45WNvzWR1Lhbk1GCQIv720CRoWvsCqfZd9eiAMRLMAAMBmQbxx2bYHOIQtLd85V/7L7AF/YsK4EcfqbQ5177jvst2D6SngMb/L1d5C5C/sD29PbfPCJfykg1YrAZHp+GnOFVAjFRc4n0sm8PeYpGYoB6ZI7uePwTpSSnOI/KyLgEUjL0Kjm6X4nGjePxZ4+xPO+DJKE3+SYGgaSqslIzJ5UoBcH+i409mjQxUrMn5Om9bXgWokswua0fNNBkoj6UrAlcl4awUoNMOxOjyNQUQn21YUDsctXTBxR9sP4mUakVoGtMQbwmDUduq/YrBeErhTbj7xmQx4hAksrlfKUNpUPyEp//thhA2IVnaDggXB/yqIU69f6tQvhvb/qbv9vx1EuGugPNFTr3+3Mc7cq2dGsyedK9pKkIj4nSH1/fi7EqEWf8jYrpr9Tjj52nBOZSb08X42cpMhhz87wzge2Q4vQTALL9TBqLBNOOIWmSJ0nFyFNoQaCd0YMQjxiuRuMZOLGYoTC5DYWFyeRhXiIdUaWP8JlhKMLbuG5CPvRgrtdWV8qXNtYBhAHwzOYdNOR6m0s2e4yigRYF/nVf/e0/Znhss4u2/O1VBkgF56f9bsWeFfDe5R7m/MHCfLW5a7RwJTjv3X2B1tj3ptw2hZLRy+Z4oO20NPgsq8MusecMPdPu+0qxurnu0BYHrv3yOcHf3BjY+DjJE+9QZPd2XuHMsXDQZxxFedCXPbd6rB/mt2n/Hex/8Bp+G7B1TbHeB38P7BnBegK/B0aBz67PSOauDuu3jQzjSLsGTJHEX7aOTNVwaLHr7NZAPsxg3QFaBcYj8JxgXw+999DPRucAfdkrCW0bhT4gjEhWpuNiKyFlDbI5Xru2jxrIiSOwo29wXh1aV+hziAJfVwqkJSTpexdfrMWnn65TzjhNDTim2CL9Z/uIra/EhEQXwrKpvkEnLVykG6+4WkfPmKOMeDy82bsNdIoEBAwWAwudvjX4BXq8MPb91eTAe+jJx3Tk3KO0r6JSv7jvAQ1Pz3VX2T5h69jWaRG651/f/HMtPvcs2ubVY489puOPnq+5Rx5FNAOvGokc9sjTT2nKrBn6dMN63Xjtz3TxWeejJSO1EbrFOJTNnaRvSgmP02lzFyjqzgj98Cc/1gkAhg899qiOnD4TfbgpoHkaB4yBaxZ6u6auTscef5yu+Y+r0Yd6ATXRY7P3GIhoy/3ic7+tVeT9q0Ovamfe9p07iR5XjPcheyDt2oGXx+133akp06ep6UCdltx2p4YZOGh7FnyAtcs8DRN5v+SCi7R7f6W2cc+by5bq7DVn6oRZ89DHEoOAMK5muTVImHJzRMkn92EjYVmTkpO18bP1Ov/sc1zo1Tj4Bkcz/rEj3cBeL+3aVLlLd917j+Yfe6y2bt6ivz79vIpoh6+XXtj+hVf4oPWZZn3/3G+pYvderVz1ifbv36+15D889ZgFTAh+/DcpdjqHSogC/2MK2ObkXgYM2V1M9gAUaOwIedJg7r2E/PAQRiQpmhj5XNLcZiEB8bSCUbAQh3WAAS+v6VZvUpQLVUHIZZdrwaz/MFRzDKghLmUNXB+Op1fTfj34H+M0sgAQaQ+eZrtgLGAu2A8AR2Aw4F1+dBwCGl503eQVhHfGuwgWeAjdsQ0qPJzng3AYKOQH9fcbsmQdYLMKbp98+Ltied0s4axtLL1Igb4hixHrhzEw
```

Comment:
I suggest we use this one which will not accept `4..5%` and will accept `4,5%` as well a `6 %`.
```
(?:\\d+([\\.,]\\d+)?\\s?%)|(?:%\\s?\\d+([\\.,]\\d+)?)
```

Reply:
This is regarding the regex in the block list to match percents.